requirements-travis.txt

```diff
@@ -16,3 +16,6 @@ psutil==3.1.1
 # stevedore for loading "new style" plugins
 stevedore==1.8.0
 lxml>=3.4.4
+# some setuptools versions can fail accessing
+# pkg_resources.packaging, let's pin the version
+setuptools==25.1.1
```

Comment:
On a second thought, looking at the current `requirements.txt` file, we don't attempt to set the development environment of the user, only to list the real requirements for Avocado.

I guess we should leave it as is at this point.  I believe that most, if not all, developers will have recent enough environments with recent enough packages.